tests/test_strategy_pvpredefined.py

```diff
@@ -20,7 +20,9 @@
 import uuid
 import pathlib
 import os
-from pendulum import DateTime, duration, today
+from pendulum import DateTime, duration, today, instance
+from datetime import date, datetime
```

Comment:
```suggestion
from pendulum import DateTime, duration, today, instance, date, datetime
```

Reply:
I agree about date and I will move it now, however this will not work with the datetime import. Reason for that is that the datetime.combine is used in the test, and as far as I know this function does not exist in pendulum. We can replace this function of course, but I would very much use the same function that the d3a_interface uses (datetime.combine)

Reply:
ok, convinced, ignore my comment then.